apps/public-docsite-v9/src/DocsComponents/FluentDocsContainer.stories.tsx

```diff
@@ -14,19 +14,27 @@ interface IFluentDocsContainerProps {
 export const FluentDocsContainer: React.FC<IFluentDocsContainerProps> = ({ children, context }) => {
   // eslint-disable-next-line deprecation/deprecation
   const selectedTheme = themes.find(theme => theme.id === context.globals[THEME_ID]);
+
```

Comment:
question(unrelated to this PR): any idea why are we naming files with `stories` suffix that are no stories at all ?